content/blog/2018/delayering-istio/delayering-istio.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Delayering Istio with AppSwitch
+description: Automatic application onboarding and latency optimizations by removing unnecessary layers with AppSwitch
+publishdate: 2018-06-28
+subtitle:
+attribution: Dinesh Subhraveti
+weight: 90
+keywords: [performance, traffic-management, appswitch]
```

Comment:
Performance, appswitch. This is not just traffic management

Reply:
In other words, this could in theory handle policies, etc. So better to keep this focused on perf and appswitch